CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+
+### Changed
```

Comment:
keep the format and add the issue number at the end of the text

Reply:
I am evil :imp:  I changed everything to my format :dark_sunglasses: